examples/complete.yaml

```diff
@@ -31,6 +31,9 @@ spec:
     - demo
     - demo2
   count: 1 # number of pods to target or a percentage (1% - 100%)
+  pulse: # optional, activate pulsing disruptions. Available for any disruptions except nodeFailure and containerFailure
```

Comment:
In addition to modifying complete.yaml, can you please add a distinct example for this feature?